docs/reference/setup/important-settings/heap-size.asciidoc

```diff
@@ -7,42 +7,48 @@ to ensure that Elasticsearch has enough heap available.
 
 Elasticsearch will assign the entire heap specified in
 <<jvm-options,jvm.options>> via the `Xms` (minimum heap size) and `Xmx` (maximum
-heap size) settings.
+heap size) settings. You should set these two settings to be equal to each
+other.
 
 The value for these setting depends on the amount of RAM available on your
-server. Good rules of thumb are:
+server:
 
-* Set the minimum heap size (`Xms`) and maximum heap size (`Xmx`) to be equal to
-  each other.
+* Set `Xmx` and `Xms` to no more than 50% of your physical RAM. {es} requires
+  memory for purposes other than the JVM heap and it is important to leave
+  space for this. For instance, {es} uses off-heap buffers for efficient
+  network communication, relies on the operating system's filesystem cache for
+  efficient access to files, and the JVM itself requires some memory too. It is
+  not unusual to observe the {es} process using more memory than the limit
```

Comment:
I think we should make a stronger statement here. Rather than "not unusual" but "normal" or "expected".

Reply:
Ok, done in 2a290bd2a6e5a27e0739485a1a322eea1740394a.